modules/nw-sriov-configuring-operator.adoc

```diff
@@ -96,6 +96,10 @@ By default, this field is set to `2`.
 |`boolean`
 |Specifies whether to enable or disable the SR-IOV Network Operator metrics. By default, this field is set to `false`.
 
+|`spec.featureGates.mellanoxFirmwareReset`
```

Comment:
This is a Tech Preview feature. Can we have a TP note here somehow?

Reply:
Thanks @cgoncalves for your feedback. I have included the standard Technology Preview text snippet that appears with all Technology Preview features.